src/app/shared/notification/notification.service.ts

```diff
@@ -32,4 +34,32 @@ export class NotificationService {
     const alert = await this.alertController.create(config);
     return await alert.present();
   }
+
+  // toast message pop up, by default, shown success message for 2 seconds.
+  async presentToast(message, success=true, duration=2000) {
+    let color = 'success'
+    if (!success) {
+      color = 'danger';
+    } 
+    const toast = await this.toastController.create({
+      message: message,
+      duration: duration,
+      position: 'top',
+      color : color
+    });
+    toast.present();
+  }
+
+  // show review rating page as pop up modal
+  // review ID is required
+  async reviewRating(reviewId, redirect) {
+     const modal = await this.modalController.create({
+      component: ReviewRatingComponent,
```

Comment:
We shouldn't import component into this service, as an alternative, we can create a generic function that accept different component in parameter
![circulardependency](https://user-images.githubusercontent.com/3895281/50676259-66c16080-102e-11e9-8b84-a75dd94f8ce2.png)